lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -11,6 +11,15 @@ const Js::OpCode LowererMD::MDExtend32Opcode = Js::OpCode::MOVSXD;
 
 extern const IRType RegTypes[RegNumCount];
 
+bool
+LowererMDArch::IsArgSaveRequired(Func *func) {
+    return (func->IsJitInDebugMode() || func->IsGeneratorFunc() ||
+        func->GetJnFunction()->GetIsAsmjsMode() || func->IsLambda() ||
```

Comment:
I see.  After my change, we can probably remove this as well then.

---
In reply to: [69825951](https://github.com/Microsoft/ChakraCore/pull/1239#discussion_r69825951) [](ancestors = 69825951)